### PR TITLE
Sync updates

### PR DESCRIPTION
**Pull request by nshahan**
Had to pin the version of sass at v1.3.2 because of a break in the way we are handling Dart package imports in style sheets.

### CODE REVIEW

**Comment by nshahan** [comment]
I filed https://github.com/sass/dart-sass/issues/334. It might be fixed by the sass package. I'm also close to releasing an update for sass_builder that will effectively avoid the issue. Not sure which one will happen first.

File: angular_gallery_section/pubspec.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: angular_gallery_section
 author: Dart Team <misc@dartlang.org>
 environment:
-  sdk: '>=2.0.0-dev.46.0 <2.0.0'
+  sdk: '>=2.0.0-dev.55.0 <2.0.0'
 dependencies:
-  analyzer: ^0.32.0
-  angular: 5.0.0-alpha+12
-  angular_components: 0.9.0-alpha+12
+  analyzer: ^0.31.1
```
[the remark this answers is not in the document]
Comment (nshahan) [reply angular_gallery_section/pubspec.yaml]: Oops, good catch! I was trying to figure out what was causing the break. That version downgrade needs to be reverted.

File: example/angular_components_example/build.yaml [hunk 2]
```diff
@@ -12,9 +12,6 @@ targets:
         enabled: True
       angular_gallery_section:
         enabled: False
-      build_web_compilers|dart2js_archive_extractor:
```
Comment (kevmoo) [inline example/angular_components_example/build.yaml]: Why these changes? @nshahan ?

Comment (nshahan) [reply example/angular_components_example/build.yaml]: Looks like a bad merge. I don't intend to remove those. I'll add them back real quick. :smiley:
